### PR TITLE
Added hostbmc api and modified the scaleiogateway string

### PR DESCRIPTION
**Pull request by venkar18**

### CODE REVIEW

File: test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py [hunk 1]
```diff
@@ -240,14 +240,21 @@ def test_cli_fru_start_quit():
...
+        child.expect("Enter hostbmc endpoint:")
+        child.sendline(af_support_tools.get_config_file_property(config_file=config_file, heading='hostbmc', property='url'))
+        child.expect("Enter hostbmc Username:")
+        child.sendline(af_support_tools.get_config_file_property(config_file=config_file, heading='hostbmc', property='username'))
+        child.expect("Enter hostbmc Password:")
+        child.sendline(af_support_tools.get_config_file_property(config_file=config_file, heading='hostbmc', property='password'))
+
         child.expect("Select a node for action \'Remove Node\':")
-        child.sendline("1")
+        child.sendline("fpr2-c6.lab.vce.com")
```
Comment (giddyelysium) [inline test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py]: Hardcoding a VM Name for Open Source? @sknightvce - this okay?

File: test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py [hunk 2]
```diff
@@ -303,14 +310,22 @@ def test_cli_fru_start():
...
+        child.sendline(af_support_tools.get_config_file_property(config_file=config_file, heading='hostbmc', property='url'))
+        child.expect("Enter hostbmc Username:")
+        child.sendline(af_support_tools.get_config_file_property(config_file=config_file, heading='hostbmc', property='username'))
+        child.expect("Enter hostbmc Password:")
+        child.sendline(af_support_tools.get_config_file_property(config_file=config_file, heading='hostbmc', property='password'))
+
         child.expect("Select a node for action \'Remove Node\':")
-        child.sendline("1")
+        # The test should take care of adding back the deleted node.
+        child.sendline("fpr2-c6.lab.vce.com")
```
Comment (giddyelysium) [inline test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py]: @sknightvce - Looking for similar advice here!

File: test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py (hunk 1 above)
Comment (venkar18) [inline test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py]: Removed the hardcoded value and replaced it with an empty string (as discussed with Colm). Changes are seen in the new commit.

File: test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py (hunk 2 above)
Comment (venkar18) [inline test-packs/test_suites/fru_paqx_parent/fru_paqx_cli/test_fru_paqx_cli.py]: Removed the hardcoded value and replaced it with an empty string (as discussed with Colm). Changes are seen in the new commit.

**Comment by giddyelysium** [comment]
@venkar18 - thanks a million - solid work

**Pull request merged**